### PR TITLE
Exercicios propostos HTML - Fernando Avellar

### PR DESCRIPTION
**Pull request by FernandoAvellar**
Olá Edy! Segue a solução dos exercícios propostos para sua avaliação!

Abraço / Fernando Avellar

### CODE REVIEW

File: fernando-avellar/1.i/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <p>O Profissional de Tecnologia da Informação tem a capacidade de compreender os problemas do mundo real, aplicação dos
+        conhecimentos adquiridos na solução de problemas, capacidade de análise sistêmica dos problemas, capacidade de síntese
+        e de realização de projetos e, finalmente, capacidade de avaliação e julgamento. As habilidades pessoais desse profissional
+        são: pensamento sistêmico, capacidade de resolução de problemas, pensamento crítico, disciplina pessoal, persistência,
+        curiosidade, capacidade de auto aprendizado e aberto a mudanças. Habilidades Interpessoais: trabalho colaborativo,
+        capacidade de comunicação, capacidade para resolução conjunta de problemas. Conhecimentos Técnicos: abstração, representação
+        e organização da informação, uso de ferramentas computacionais para aplicação dos conhecimentos.</p>
+
+
+    <h3>Corpo Docente</h3>
```
Comment (edysegura) [inline fernando-avellar/1.i/index.html]: Why did you choose to use `<h3>` here instead of `<h2>`?

File: fernando-avellar/1.i/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <li>Elton Barbosa</li>
+        <li>Felipe Andery</li>
+        <li>Márcio Emílio</li>
+        <li>Paulo Siécola</li>
+        <li>Phyllipe Lima</li>
+        <li>Roberto Rocha</li>
+        <li>Rodrigo Pimenta</li>
+    </ul>
+
+    <h4>Informações</h4>
```
Comment (edysegura) [inline fernando-avellar/1.i/index.html]: Same question here

File: fernando-avellar/1.ii/index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    </ul>
+
+    <h2>Idiomas</h2>
+    <ul>
+        <li>Inglês: Fluente</li>
+        <li>Espanhol: Fluente</li>
+        <li>Português: Nativo</li>
+    </ul>
+
+    </ul>
```
Comment (edysegura) [inline fernando-avellar/1.ii/index.html]: My guess is that you forgot this last `</ul>` in your document. Am I right?

File: fernando-avellar/1.ii/index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline fernando-avellar/1.ii/index.html]: As this document is in portuguese language we need to update the attribute lang to "pt-br"

File: fernando-avellar/1.iii/index.html [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+        <li><a href="#introducao">Introducao</a></li>
+        <li><a href="#capitulo1">Capítulo 1</a></li>
+        <li><a href="#capitulo2">Capítulo 2</a></li>
+        <li><a href="#capitulo3">Capítulo 3</a></li>
+        <li><a href="#apendicea">Apêndice A</a></li>
+        <li><a href="#bibliografia">Referências Bibliográficas</a></li>
+    </ul>
+
+    <a name="agradecimentos"></a>
+    <h2>Agradecimentos</h2>
```
Comment (edysegura) [inline fernando-avellar/1.iii/index.html]: Instead of use `<a name="agradecimentos"></a>` try `<h2 id="agradecimentos">Agradecimentos</h2>`

**Review by edysegura — COMMENTED** [review]
Hi @FernandoAvellar !

Please let me know if you have any doubts about my comments.

Cheers,

**Review by FernandoAvellar — COMMENTED** [review]
Hi Edy! I have done the requested changes!  Waiting for your comments! Thanks.

**Pull request merged**